### ISSUE TITLE
Color Contrast Accessibility issue when using default SkyUX Avatar component

### ISSUE THREAD

**Issue by Blackbaud-markarita** [blackbaud/skyux-avatar#14]
### Expected behavior
No error is presented with default avatar

### Actual behavior
When running a test for accessibility with a page that includes the Avatar component, a color contrast error is presented

aXe - [Rule: 'color-contrast'] Elements must have sufficient color contrast - WCAG: wcag143
Get help at: https://dequeuniversity.com/rules/axe/2.6/color-contrast?application=axeAPI
Elements:
<div _ngcontent-a-c2943="" class="sky-avatar-initials-inner"> T1 </div>

### Steps to reproduce
HTML code
<sky-avatar [canChange]=false
[name]=sg.name
[src]=sg.avatar>
</sky-avatar>

Test
it('should be accessible', async(() => {
setupPermissions();
setupInit(1);
fixture.detectChanges();
fixture.whenStable().then(() => {
const page = fixture.debugElement.nativeElement.querySelector('section');
expect(page).toBeAccessible();
});
}));

### Severity
Low

### Impact
Low

### Workaround
Update test to be:

      const page = fixture.debugElement.nativeElement.querySelector('section');
      expect(page).toBeAccessible(() => { }, {
        rules: {
        'color-contrast': { enabled: false }
        }
      });

**Comment by Blackbaud-TrevorBurch** [comment]
We did have another consumer report of this on 9/17/2020
